main.js

```diff
@@ -33,6 +33,7 @@ class Parcel extends utils.Adapter {
         this.sessions = {};
         this.mergedJson = [];
         this.inDelivery = [];
+        this.delivered = [];
```

Comment:
The plan is to refactor this to a new property
short_status
with states

IN_TRANSIT
OUT_FOR_DELIVERY
DELIVERED

Reply:
Würde **delivery_status** statt **short_status** besser finden. Da ist dann der Bezug vorhanden um welchen Status es sich handelt.

Die Werte sollten lediglich als Konstanten benannt sein und als Integer gespeichert werden, damit eine einfache Sortierung möglich ist. Es sollten auch "Lücken" existieren, um falls nötig ergänzen zu können, ohne den ganzen nachfolgenden Code (Widget) kaputt zu machen.

10 - OUT_FOR_DELIVERY
20 - IN_TRANSIT
30 - IN_PREPARATION
40 - DELIVERED

Dann kann einfach sortiert / gefiltert werden und wenn nochmal ein Status dazu kommt kann er ohne Probleme eingebaut werden.

Ich hoffe es ist Dir Recht wenn ich da ein wenig mitmache. Mich interessiert das Thema Adapter und wenn ich Dir nicht zu viel Arbeit damit mache, dann hoffe ich, den ein oder anderen hilfreichen Beitrag leisten zu können.
Vielleicht noch kurz zu meinem Hintergrund:
Ich bin Technischer Leiter in der IT, habe etliche Jahre Programmiererfahrung (hauptsächlich C#) hinter mir und nutze ioBroker schon seit ein paar Jahren. Bisher nur ein "normaler" User, will ich so langsam auch meinen Beitrag zu dem Projekt leisten ;)

Reply:
Ok dann merge ich das jetzt und mach noch heute anpassung auf den du dann weiter arbeiten kannst

Reply:
Habe ein delivery state eingefügt